be/src/exec/pipeline/pipeline_driver.cpp

```diff
@@ -602,9 +602,21 @@ void PipelineDriver::_adjust_memory_usage(RuntimeState* state, MemTracker* track
         }
         request_reserved += state->spill_mem_table_num() * state->spill_mem_table_size();
 
+        bool need_spill = false;
```

Comment:
[[maybe_unused]]

Reply:
it won't generate a WARNING. TRACE_LOG is VLOG 
```
if (LOG_ENALE_LEVEL(MODULE)) {
 VLOG(LEVEL) << xxx
}
```